website/docs/reference/resource-configs/database.md

```diff
@@ -2,38 +2,85 @@
 sidebar_label: "database"
 resource_types: [models, seeds, tests]
 datatype: string
-description: "Read this guide to understand the database configuration in dbt."
+description: "Override the default database when dbt creates resources in your data platform."
 ---
 
-:::caution Heads up!
-This is a work in progress document. While this configuration applies to multiple resource types, the documentation has only been written for seeds.
+<Tabs>
+<TabItem value="model" label="Model">
 
-:::
+Specify a custom database for a model in your `dbt_project.yml` file. 
 
-## Definition
+For example, if you have a model that you want to load into a database other than the target database, you can configure it like this:
 
-Optionally specify a custom database for a [model](/docs/build/sql-models) or [seed](/docs/build/seeds). (To specify a database for a [snapshot](/docs/build/snapshots), use the [`target_database` config](/reference/resource-configs/target_database)).
+<File name='dbt_project.yml'>
 
-When dbt creates a relation (<Term id="table" />/<Term id="view" />) in a database, it creates it as: `{{ database }}.{{ schema }}.{{ identifier }}`, e.g. `analytics.finance.payments`
+```yml
+models:
+  your_project:
+    sales_metrics:
+      +database: reporting
+```
+</File>
 
-The standard behavior of dbt is:
-* If a custom database is _not_ specified, the database of the relation is the target database (`{{ target.database }}`).
-* If a custom database is specified, the database of the relation is the `{{ database }}` value.
+This would result in the generated relation being located in the `reporting` database, so the full relation name would be `reporting.finance.sales_metrics` instead of the default target database.
+</TabItem>
 
-To learn more about changing the way that dbt generates a relation's `database`, read [Using Custom Databases](/docs/build/custom-databases)
+<TabItem value="seeds" label="Seeds">
+
+Configure a database in your `dbt_project.yml` file. 
 
-## Usage
+For example, to load a seed into a database called `staging` instead of the target database, you can configure it like this:
 
-### Load seeds into the RAW database
 <File name='dbt_project.yml'>
 
 ```yml
 seeds:
-  +database: RAW
+  your_project:
+    product_categories:
+      +database: staging
+```
+
+This would result in the generated relation being located in the `staging` database, so the full relation name would be `staging.finance.product_categories_data`.
```

Comment:
Where did the `_data` suffix come from here?

Reply:
oh fudge! i think i copied the <tabs> over from alias and didnt rmove them